com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -296,6 +296,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Splited HDProjectSettings with new HDUserSettings in UserProject. Now Wizard working variable should not bother versioning tool anymore (case 1330640)
 - Removed redundant Show Inactive Objects and Isolate Selection checkboxes from the Emissive Materials tab of the Light Explorer (case 1331750).
 - Renaming Decal Projector to HDRP Decal Projector.
+- The HDRP Render Graph now uses the new RendererList API for rendering and (optional) pass culling.
```

Comment:
looks like there is a merge issue with the changelog

Reply:
@sebastienlagarde do you mean these lines shouldn't appear in my PR? I think if you look at the changes in the merge commit itself, it displays "unrelated changes" that happened in that merge. But if you view [this file in "Show all commits" view](https://github.com/Unity-Technologies/Graphics/pull/4616/files?file-filters%5B%5D=.md#diff-621f39c820c4b8b22426a93fb942adb48b98d3955ec07eec68d40a1726bea33f), it correctly displays only the one line that I've added.